packages/astro/src/core/errors/errors-data.ts

```diff
@@ -1309,6 +1309,20 @@ export const CannotExtractFontType = {
 	hint: 'Open an issue at https://github.com/withastro/astro/issues.',
 } satisfies ErrorData;
 
+/**
+ * @docs
+ * @description
+ * Cannot extract the font data from the given URL.
+ * @message
+ * An error occured while trying to extract the font data from the given URL.
+ */
+export const CannotExtractFontData = {
+	name: 'CannotExtractFontType',
+	title: 'Cannot extract the font data from the given URL.',
+	message: (family: string, url: string) => `An error occurred while trying to extract the font data from local font family "${family}" (url: ${url})`,
+	hint: 'Update your family config and disable weight and style inference. You must set the value manually instead.',
```

Comment:
Rather than talking abotu disabling inference I'd just say to set it manually, because that's how you disable inference.

Reply:
```suggestion
	hint: 'Update your family config and set \`weight\` and \`style\` manually instead.',
```